src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -3030,6 +3030,43 @@ def test_aks_nodepool_add_with_ossku_azurelinux3(self, resource_group, resource_
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus2euap')
+    def test_aks_nodepool_add_with_ossku_flatcar(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=138566135&view=results) to validate the change.